proceeding/text/features.tex

```diff
@@ -37,11 +35,19 @@ \section{Features}
 
   \end{itemize}
 
-  (Figure of Catalog View, 3FGL source with Light Curve and Emission Spectrum)
-
 
 \item Further analysis of our data with tools like Gammapy (generate specific plots, etc.)
 
 \item Mention again that all data is openly available for download
 
 \end{enumerate}
+
+\begin{figure}[h]
+  \centerline{\includegraphics[width=\textwidth]{figures/mapview_wide}}
+  \caption{Map View.}
+\end{figure}
+
+\begin{figure}[h]
+  \centerline{\includegraphics[width=\textwidth]{figures/catview_wide}}
+  \caption{Catalog View.}
```

Comment:
The catalog view image is a bit small. Remove white space on left and right so that the content is better visible?